### PR TITLE
Combine all services onto one page and host bundle.js files on s3

### PR DESCRIPTION
**Pull request by vvek87**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,13 +5,17 @@
...
+    <div id="descriptions"></div>
+    <div id="reviews"></div>
+    <div id="morehomes"></div>
 
     <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
     <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
 
-    <script src="http://localhost:3004/bundle.js"></script>
-    <script src="http://localhost:3000/bundle.js"></script>
+    <script src="https://s3-us-west-1.amazonaws.com/fecbundlefiles/top+images/bundle.js"></script>
```
Comment (mikeglopez) [inline public/index.html:16]: What is the point in hosting the bundles? Is this just so you don't have to manually start the servers?

Comment (mikeglopez) [reply public/index.html:16]: Do you need all the services as submodules in your repo if all you're using from them is their bundle file?

**Review by mikeglopez — APPROVED** [review]
Looks good, Vivek! Just a couple of comments to think about, but do whatever works :)

File: public/index.html (hunk 1 above)
Comment (vvek87) [reply public/index.html:16]: I'm hosting them according to the description on Learn under project_milestones, but without the server files for each service, I'm having an issue with the GET requests not being sent to the specified routes on the specified ports.

**Pull request merged**